grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcUnsafeBufferUtil.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.grpc;
```

Comment:
Wondering if this class should be in `internal`. What do you think, @anuraaga ?

Reply:
It's an interesting question - I put `ByteBufHttpData` in internal since I wasn't really expecting it to be used outside of armeria, but maybe it is reasonable to expect users to use it. On the flip side, this class has to be used by users no matter what.

I was thinking of the `internal` package as one that wouldn't be exported if using Java9 modules - if that's true, I think this location for `GrpcUnsafeBufferUtil` is correct, but we may want to reconsider whether `ByteBufHttpData` deserves to be exported or not.

Reply:
Yes, how about introducing a new package `com.linecorp.armeria.unsafe`? `ByteBufHttpData` could be moved there and `GrpcUnsafeBufferUtil` could be to `com.linecorp.armeria.unsafe.grpc`?

Reply:
Good idea - moved to unsafe package